src/modules/organisation/api/organisation.uc.spec.ts

```diff
@@ -83,4 +87,51 @@ describe('OrganisationUc', () => {
             );
         });
     });
+
+    describe('findAll', () => {
```

Comment:
Test structure, the when clause is missing.

Reply:
✅